service-base/src/main/java/org/eclipse/hono/service/registration/RegistrationAmqpEndpoint.java

```diff
@@ -33,7 +33,7 @@
  * It receives AMQP 1.0 messages representing requests and sends them to an address on the vertx
  * event bus for processing. The outcome is then returned to the peer in a response message.
  */
-public final class RegistrationAmqpEndpoint extends RequestResponseEndpoint<ServiceConfigProperties> {
+public class RegistrationAmqpEndpoint extends RequestResponseEndpoint<ServiceConfigProperties> {
```

Comment:
Please make the `getName()` method final, too.

Reply:
No. It was intentional so that overriding classes can give a custom name.

Reply:
Done